### PR TITLE
Add tooluse to SaaS

### PR DESCRIPTION
**Pull request by MichaelDeng03**
Add crud util for tools
Add name + user_id as fields and a constraint for Tools
Add tools to SaaS
Remove demo/server.py

### CODE REVIEW

**Comment by kgang** [comment]
I'll just leave a pre-emptive approve

**Review by kgang — APPROVED** [review]

**Comment by MichaelDeng03** [comment]
i don't think i'm gonna be able to get this merged in before the next demo

**Comment by abjjabjj** [comment]
all g

File: webapp/main/server.py [hunk 1]
```diff
@@ -490,12 +522,13 @@ async def project_chat_ws(websocket: WebSocket, orchestrator_id: UUID, project_i
                     raise HTTPException(status_code=404, detail=f"Developer {project.executive_id} not found")
                 executive = sqlmodel_executive.to_obj()
                 executive.project_id = project.id
-
+                print(executive.tools)
```
Comment (abjjabjj) [inline webapp/main/server.py:525]: `CliClient.emit`

File: webapp/main/server.py [hunk 2]
```diff
@@ -490,12 +522,13 @@ async def project_chat_ws(websocket: WebSocket, orchestrator_id: UUID, project_i
...
                 sqlmodel_developer = crud.get_operator(session, project.developer_id, orchestrator_id)
                 if sqlmodel_developer is None:
                     raise HTTPException(status_code=404, detail=f"Developer {project.developer_id} not found")
                 developer = sqlmodel_developer.to_obj()
                 developer.project_id = project.id
+                print(developer.tools)
```
Comment (abjjabjj) [inline webapp/main/server.py:531]: `CliClient.emit`

File: webapp/main/server.py [hunk 3]
```diff
@@ -313,8 +341,12 @@ def db_getter():
...
 
 @app.get("/orchestrators/{orchestrator_id}/operators/form", response_class=HTMLResponse)
-async def create_operator_form(orchestrator_id: UUID, request: Request):
-    return sidebar_create_operator(orchestrator_id, request)
+async def create_operator_form(orchestrator_id: UUID, request: Request, user_id: UserIdDep):
+    with Session() as session:
+        tools = crud.get_user_tools(session, user_id)
+        tool_names = [tool.name for tool in tools]
+
+    return sidebar_create_operator(orchestrator_id, request, tool_names)
```
Comment (abjjabjj) [inline webapp/main/server.py:349]: TODO to pass entire tool to FE. saves us DB calls in `create_operator`

**Review by abjjabjj — APPROVED** [review]

**Pull request merged**